test/runtests.jl

```diff
@@ -5,5 +5,7 @@ include("test_misc.jl")
 include("test_ops.jl")
 include("test_ad.jl")
 
-# Build the docs
```

Comment:
Hmm lets do like this instead:

in `make.jl` change `strict = true` to `strict = VERSION.minor == 6`  or something.